### PR TITLE
Upp factset provisioner

### PR DESCRIPTION
**Pull request by scott-ace-newton**

### CODE REVIEW

File: upp-factset-provisioner/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Provisioning a cluster
+
+The provisioning process will:
+
+* Creates an aurora cluster, db subnet group and cluster parameter group in eu-west-1 using the cloud formation template
+
+How to run:
+
+# Parameters
```
Comment (peteclark-ft) [inline upp-factset-provisioner/README.md]: I think the TAG_ENVIRONMENT should be included here (i.e. `t` or `p`)

File: upp-factset-provisioner/sh/provision.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo $VAULT_PASS > /ansible/vault.pass
+
+cd /ansible
+
+ansible-playbook --vault-password-file=vault.pass provision.yml --extra-vars "\
```
Comment (peteclark-ft) [inline upp-factset-provisioner/sh/provision.sh]: Similarly, we should probably be passing through the `tag_environment` here

File: upp-factset-provisioner/README.md (hunk 1 above)
Comment (scott-ace-newton) [reply upp-factset-provisioner/README.md]: I originally had it as a parameter to be passed in but instead opted to include it in the content-test and content-prod ansible files as each account will only create t or p instances

File: upp-factset-provisioner/sh/provision.sh (hunk 2 above)
Comment (scott-ace-newton) [reply upp-factset-provisioner/sh/provision.sh]: same as above

**Comment by scott-ace-newton** [comment]
I removed the parameter group stuff as we are only creating a single instance. Thus no need for separate instance config; the cluster parameter group config should suffice

File: upp-factset-provisioner/README.md (hunk 1 above)
Comment (peteclark-ft) [reply upp-factset-provisioner/README.md]: Ohh is it in the ansible vault files?

File: upp-factset-provisioner/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+* VAULT_PASS: Used to read ansible provisioning data, can be found in last pass under upp-factset-provisioner
+* AWS_ACCOUNT: Account in which to provision RDS; must be either content-test or content-prod
+
+```
+docker run   \
+    -e "MASTER_PASSWORD=$MASTER_PASSWORD" \
+    -e "ENVIRONMENT_NAME=$ENVIRONMENT_NAME" \
+    -e "VAULT_PASS=$VAULT_PASS" \
+    -e "AWS_ACCOUNT=$AWS_ACCOUNT" \
+    coco/upp-factset-provisioner:local /bin/bash provision.sh
```
Comment (JermilaPaulDhas) [inline upp-factset-provisioner/README.md]: the name of the image does not match with the name in the build command. It need to be `docker build -t upp-factset-provisioner:local .`

File: upp-factset-provisioner/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+- Get the environment variables from the **User upp-factset-provisioner** LastPass note in the **Shared-UPP Credentials & Services Login Details** folder.
+- Run the following docker command
+
+```
+docker run   \
+    -e "ENVIRONMENT_NAME=$ENVIRONMENT_NAME" \
+    -e "VAULT_PASS=$VAULT_PASS" \
+    -e "AWS_ACCOUNT=$AWS_ACCOUNT" \
+    coco/upp-factset-provisioner:local /bin/bash decom.sh
```
Comment (JermilaPaulDhas) [inline upp-factset-provisioner/README.md]: same applies here too

File: upp-factset-provisioner/README.md (hunk 1 above)
Comment (peteclark-ft) [reply upp-factset-provisioner/README.md]: The LastPass note doesn't contain the vault pass, can you add to that?

Comment (scott-ace-newton) [reply upp-factset-provisioner/README.md]: I havent actually made a folder yet and almost went on holiday thanks :)

**Comment by scott-ace-newton** [comment]
This is the first ive heard about using `d` for team environments, if so then it will need to be removed from the vault and added as a passed in param

**Comment by scott-ace-newton** [comment]
moved to a new pr https://github.com/Financial-Times/upp-provisioners/pull/136

**Pull request closed without merging**